### DISCUSSION TITLE
Collections: Citation Title vs. Display Title

### DISCUSSION THREAD

**Discussion by kyledenlinger** [knowledgefutures/pubpub#2098] [Ideas]
It'd be cool if collections could be given two names that are used in different spots:
- The Full "Citation Title" for the collection's page, citation metadata, etc.
- An optional "Display Title" that would show up on the collection/tag area at the top of a pub, on a collection tile in a Collections & Pages block, etc. 

Use case: say a citation title is super long and takes up too much real estate in the tag area or looks weird in a collection preview image. A display title could be an abbreviated version of the full title that allows other collections & tags to be exposed.

**Comment by katycroffbell** [comment]
Agree! This would be great.
